packages/amplify-provider-awscloudformation/src/__tests__/iterative-deployment/helper.test.ts

```diff
@@ -0,0 +1,216 @@
+import * as helper from '../../iterative-deployment/helpers';
+
+import { DeployMachineContext, DeploymentMachineOp } from '../../iterative-deployment/state-machine';
+
+describe('deployment helpers', () => {
+  const baseDeploymentOp: DeploymentMachineOp = {
+    parameters: {
+      deploying: 'true',
+    },
+    region: 'us-east-2',
+    stackName: 'my-stack',
+    stackTemplatePath: 'stackTemplatePath',
+    stackTemplateUrl: 'stackTemplateUrl',
+    tableNames: ['table1'],
+    capabilities: [],
+  };
+
+  const deploymentContext: DeployMachineContext = {
+    currentIndex: -1,
+    deploymentBucket: 'my-bucket',
+    region: 'us-east-2',
+    stacks: [
+      {
+        deployment: {
+          ...baseDeploymentOp,
+        },
+        rollback: {
+          ...baseDeploymentOp,
+          parameters: { rollingBack: 'true' },
+        },
+      },
+    ],
+  };
+
+  describe('isDeploymentComplete', () => {
+    it('should return false when currentIndex is smaller than deployment steps', () => {
+      const context: DeployMachineContext = {
+        ...deploymentContext,
+        currentIndex: -1,
+      };
+      expect(helper.isDeploymentComplete(context)).toBeFalsy();
```

Comment:
nit: I prefer to use `.toBe(false)` or `.toBe(true)` when a boolean is expected.